src/OrbitCaptureClient/CaptureClient.cpp

```diff
@@ -58,6 +58,7 @@ static CaptureOptions::InstrumentedFunction::FunctionType InstrumentedFunctionTy
         FunctionInfo_OrbitType_FunctionInfo_OrbitType_INT_MAX_SENTINEL_DO_NOT_USE_:
       UNREACHABLE();
   }
+  UNREACHABLE();
```

Comment:
Why not just restore the `default` in the switch? I don't think we should use those values directly.

Reply:
because we want a compile time error when enum class changes.

Reply:
But with the addition of this `UNREACHABLE()` would you still get that compile time error now? If I get this correctly this new `UNREACHABLE()` acts like `default` now.

Reply:
yes, if there is no default in the switch - and you add another value to enum you get compile time error. (it is a warning promoted to a error)

Reply:
Makes sense.